Makefile

```diff
@@ -50,6 +50,13 @@ test-api:
 
 	# get lpas
 	./api-test/tester -expectedStatus=200 REQUEST POST $(URL)/lpas '{"uids": [$(LPA_UID)]}'
+
+	# certificate provider opt out
+	$(eval LPA_UID := "$(shell ./api-test/tester UID)")
+	cat ./docs/example-lpa.json | ./api-test/tester -expectedStatus=201 REQUEST PUT $(URL)/lpas/$(LPA_UID) "`xargs -0`"
+	./api-test/tester -expectedStatus=200 -write REQUEST GET $(URL)/lpas/$(LPA_UID) '' > $(TMPFILE)
```

Comment:
you aren't using the TMPFILE after, so could remove `-write` and `> $(TMPFILE)`

Reply:
Ah hadn't twigged what was happening there. Thanks, I'll remove 👍